### PR TITLE
Added more linux commands

### PR DESCRIPTION
**Pull request by shahbaz17**
https://duck.co/ia/view/linux_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by shahbaz17** [comment]
Hello @moollaza Can you look into this?
Why there is an error ? Can anyone here explain

File: share/goodie/cheat_sheets/json/linux.json [hunk 1]
```diff
@@ -8,19 +8,20 @@
...
         "Process Management",
+        "Shortcuts",
         "File Permissions",
-        "Compression",
-        "Shortcuts"
+        "Compression"
     ],
     "sections":{
-        "File Commands":[
+        "Navigation":[
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/linux.json]: @shahbaz17 Put a space between colons and values (`x: y`).

Comment (shahbaz17) [inline share/goodie/cheat_sheets/json/linux.json]: @GuiltyDolphin Can you look again? Is this ok

File: share/goodie/cheat_sheets/json/linux.json [hunk 2]
```diff
@@ -1,379 +1,430 @@
...
             },
             {
-                "val":"decompress file.gz",
-                "key":"gzip -d file.gz"
+                "val": "decompress file.gz",
+                "key": "gzip -d file.gz"
             }
         ],
-        "Shortcuts":[
+        "Shortcuts": [
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/linux.json]: Lets lose this section - it's not really appropriate to the cheat sheet (nor does it display well, given we are using the `terminal` template type).

**Comment by shahbaz17** [comment]
@GuiltyDolphin So i need to remove Shortcut section ? Right ??

File: share/goodie/cheat_sheets/json/linux.json [hunk 3]
```diff
@@ -1,379 +1,327 @@
...
             {
-                "val":"show memory and swap usage",
-                "key":"free"
+                "val": "display who is online",
+                "key": "w"
             },
             {
-                "val":"show the location of the executable 'app'",
-                "key":"whereis app"
+                "val": "who are you logged in as",
```
Comment (MrChrisW) [inline share/goodie/cheat_sheets/json/linux.json]: `display current user`

File: share/goodie/cheat_sheets/json/linux.json [hunk 4]
```diff
@@ -1,379 +1,327 @@
...
             {
-                "val":"extract",
-                "key":"x"
+                "val": "display currrently active processes",
+                "key": "ps"
             },
             {
-                "val":"specifies filename",
-                "key":"f"
+                "val": "ps with a lot of details",
```
Comment (MrChrisW) [inline share/goodie/cheat_sheets/json/linux.json]: Maybe `show processes for all users` would be better.

http://unix.stackexchange.com/questions/106847

File: share/goodie/cheat_sheets/json/linux.json [hunk 5]
```diff
@@ -1,379 +1,327 @@
...
             {
-                "val":"erases whole line",
-                "key":"[ctrl] [u]"
+                "val": "untar into current directory",
+                "key": "tar xf file.tar"
             },
             {
-                "val":"reverse lookup of previous commands",
-                "key":"[Ctrl] [r]"
+                "val": "show contents of archive",
```
Comment (MrChrisW) [inline share/goodie/cheat_sheets/json/linux.json]: `list the contents of an archive`

File: share/goodie/cheat_sheets/json/linux.json [hunk 6]
```diff
@@ -1,379 +1,327 @@
 {
-    "id":"linux_cheat_sheet",
-    "name":"Linux",
+    "id": "linux_cheat_sheet",
+    "name": "Linux",
     "description": "Linux is an open-source Unix-like operating system dveloped by Linus Torvalds",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/linux.json]: Perhaps just `An open-source Unix-like operating system developed by Linus Torvalds`.

**Comment by GuiltyDolphin** [comment]
@shahbaz17 Looks like you've got yourself a case of ***merge conflicts***! Might want to get that sorted ;)

**Comment by shahbaz17** [comment]
@GuiltyDolphin How to fix this? Since few days back it was all good to be merged. And now showing merge conflict...

**Comment by shahbaz17** [comment]
Closing this PR since started a new one here https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2400

@GuiltyDolphin Help needed in the above PR if something went wrong.

**Pull request closed without merging**